source/Lib/DecoderLib/DecLibRecon.cpp

```diff
@@ -729,15 +729,18 @@ bool DecLibRecon::ctuTask( int tid, CtuTaskParam* param )
       {
         CtuData &ctuData = cs.getCtuData( ctu, line );
         ctuData.motion = decLib.m_ctuDataBufs[ctu + line * cs.pcv->widthInCtus].motion;
-        GCC_WARNING_DISABLE_class_memaccess
-        memset( ctuData.motion, 0, sizeof( CtuDataBuffers::motion ) );
-        GCC_WARNING_RESET
```

Comment:
The idea behind that is, that setting a large block of continous memory to '0' should be faster, than setting a lot of small non-continous blocks (even if only, say.. 1/4 of the actual memory needs to be reset).

Did you test this specific change for performance?

Reply:
In most scenes, the proportion of inter mode is larger than that of intra mode. This optimization can save a memset in inter mode.

For the 0/2/10/20 thread configuration, the average decoding time is 99.53%/99.14%/99.78%/100.06% as before;